src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ExternalUserEntity.kt

```diff
@@ -27,6 +27,6 @@ data class ExternalUserEntity(
 
   @CreationTimestamp
   private val createdAt: OffsetDateTime? = null,
-) {
+) : Cas2User() {
```

Comment:
first time using sealed classes like this, interested in any feedback. It seemed neater than introducing e.g. a new table that joins nomis/external users table

Reply:
I agree, a join table would just add so much more complexity that this approach